README.md

```diff
@@ -1,46 +1,99 @@
+  - `git clone https://github.com/MrOmachi/Crypto-Metrics.git `
+  - Open the folder "crypto_metrics
+    and run `npm install` to install all necesary scripts,
+  - Run from your termminal, `npm start` to open the project in your browser.
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+**This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).**
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+To get a local copy up and running follow these simple example steps.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+### Prerequisites
 
-## Learn More
+- [NodeJS](https://nodejs.org/en/docs/)
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+> As an asynchronous event-driven JavaScript runtime, Node.js is designed to build scalable network applications. In the following "hello world" example, many connections can be handled concurrently. Upon each connection, the callback is fired, but if there is no work to be done, Node.js will sleep.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- [Text Editor(VScode Recommended)](https://code.visualstudio.com/)
+
+> Visual Studio Code is a free coding editor that helps you start coding quickly. Use it to code in any programming language, without switching editors.
+
+### Setup
+
+**Run from your terminal:**
+
+- `npx create-react-app ./`
+
+**cd into space-traveler-hub folder, then run from your terminal:**
+
+- `npm start`
+
+### Usage
+
+**When the above commands aresuccessfully run, your browser should open on `localhost:3000`, if not, open your browser and load `127.0.0.1:3000` to see your default template loaded**
+
+### Run tests
+
+> coming soon :smile:
+
+### Deployment
+
+> coming soon :smile:
+
+## Authors
+
+👤 _Omachi John_
+
+- GitHub: [@githubhandle](https://github.com/MrOmachi)
+- Twitter: [@twitterhandle](https://twitter.com/Mr_Omachi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/john-omachi-00446210b)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] please make sure to add credit to the author of the design as in the requirements 
<img width="1105" alt="Screen Shot 2022-10-05 at 16 12 20" src="https://user-images.githubusercontent.com/61790587/194096381-1294b887-b8e8-4339-9a3e-f8c198029daa.png">

Reply:
Template owner was acknowledged.